core/arch/x86/mangle.c

```diff
@@ -533,11 +534,11 @@ insert_pop_all_registers(dcontext_t *dcontext, clean_call_info_t *cci, instrlist
     }
 
     PRE(ilist, instr,
-        INSTR_CREATE_lea(dcontext, opnd_create_reg(REG_XSP),
-                         OPND_CREATE_MEM_lea(REG_XSP, REG_NULL, 0,
-                                             PRE_XMM_PADDING +
-                                                 MCXT_TOTAL_SIMD_SLOTS_SIZE +
-                                                 offs_beyond_xmm)));
+        INSTR_CREATE_lea(
+            dcontext, opnd_create_reg(REG_XSP),
+            OPND_CREATE_MEM_lea(REG_XSP, REG_NULL, 0,
+                                PRE_XMM_PADDING + MCXT_TOTAL_SIMD_SLOTS_SIZE +
+                                    MCXT_TOTAL_OPMASK_SLOTS_SIZE + offs_beyond_xmm)));
```

Comment:
I think a FIXME/TODO comment is required making it clear that filling these values in is forthcoming, b/c o/w this leaves the code in a weird state where it accounts for space for these registers but leaves them uninitialized and thus looks incorrect.

Reply:
Ok. As above, comments added.